modules/aws/hub/variables.tf

```diff
@@ -150,6 +155,12 @@ variable "binaries_location" {
   nullable    = false
 }
 
+variable "tarball_url" {
+  type        = string
+  default     = ""
+  description = "https DSF installation location, defaults to using binaries_location"
```

Comment:
Please change it to:
HTTPS DSF installation location. If not set, binaries_location is used

Change binaries_location description to:
S3 DSF installation location. If tarball_url not set, binaries_location is used

Change at all modules: hub, GW and base instance